public/js/whiteboard.js

```diff
@@ -6,7 +6,7 @@ const latitude = 42.72927;
 //Time Constants(in Milliseconds)
 const oneMinute = 60000;
 const thirtyMinutes = oneMinute * 30;
-const threeMinutes = oneMinute * 3;
+const threeMinutes = oneMinute * 5;
```

Comment:
Except `threeMinutes` shouldn't equal 5

Maybe do `timeoutTime` and make that configurable.

Reply:
I was hoping you wouldn't notice...

Reply:
Should I make it configurable in the environment variable, or is what I've done acceptable?